src/types/index.ts

```diff
@@ -1,9 +1,65 @@
 // @ts-ignore
 interface AutoComplete {}
 
-interface StopGroup {}
+interface StopGroup {
+  country_code: string;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: string;
+  childs: SegmentProviderStop[];
+  serviced?: string;
+  has_been_modified?: boolean;
+  warning?: boolean;
+  is_latest?: boolean;
+}
+
+interface StopCluster {
+  country_code: string;
+  name: string;
+  longitude: number;
+  latitude: number;
+  type: string;
+  childs: string[];
+  unique_name?: string;
+  serviced?: string;
+  has_been_modified?: boolean;
+  warning?: boolean;
+  is_latest?: boolean;
+}
 
-interface StopCluster {}
+type GroundPlacesList = {
+  [key: string]: StopGroup | StopCluster;
+};
+
+interface StopInfos {
```

Comment:
What is a stop ?
Is this a SegmentProviderStop ?

Reply:
This is the Infos of a place when we want to create a new one (StopGroup or StopCluster), but regards to the spec, this is not specific to a type of place, I'll update this

Reply:
Done